### PR TITLE
Upgrade 

### PR DESCRIPTION
**Pull request by ngarg-vail**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 None
 
+<a name="4.0.0"></a>
+## [4.0.0] - 2022-03-03
+### Changed
+- Prefer keyword argument initialization for all models
+- Rename api->list_an_application to list->list_applications
+- Use new openapi-generator version (5.4.0)
+- Remove floating `Freeclimb::percl_to_json` and added it to `PerclScript` as `to_json` method
```
Comment (wajiw) [inline CHANGELOG.md:16]: Looks like the quickstarts are using percl_to_json so make sure to create tickets to update the docs to use the new to_json

**Review by wajiw — APPROVED** [review]
Everything looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (ngarg-vail) [reply CHANGELOG.md:16]: updated quickstarts: https://github.com/FreeClimbAPI/Ruby-Getting-Started-Tutorial/pull/3/files

**Comment by wajiw** [comment]
+1 @ngarg-vail

**Pull request merged**